controlplane/src/core/services/Keycloak.ts

```diff
@@ -66,10 +75,60 @@ export default class Keycloak {
           temporary: isPasswordTemp,
         },
       ],
+      id,
     });
     return createUserResp.id;
   }
 
+  public async updateKeycloakUser({
+    realm,
+    id,
+    password,
+    firstName,
+    lastName,
+    groups,
+    enabled,
+  }: {
+    id: string;
+    realm?: string;
+    password?: string;
+    firstName?: string;
+    lastName?: string;
+    groups?: string[];
+    enabled?: boolean;
```

Comment:
If a user was disabled should be visible in the UI.

Reply:
Done